bundles/org.openhab.binding.casokitchen/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="casokitchen"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current Zone Temperature</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
```

Comment:
When adding tags: i would expect Temperature and Measurement here

Reply:
Just one concern 
 I don't have a room  temperature device in my kitchen. So the model is showing the fridge temp as default. If this is ok I'm fine!
![image](https://github.com/user-attachments/assets/b08ce8d0-40e4-4962-95a5-9e9a5f137e37)

Reply:
It is still a measurement of a temperature. It is not restricted to a room. So that will be fine

Reply:
Perhaps the system channel type `system:indoor-temperature` could be used here with overridden label and description? See for example #18325.

Reply:
The ‘indoor’ part of that channel type suggests it is about a room / indoor location.
This temperature is for the zone within the fridge, I dont’t think that channel type is a good match here.

Reply:
@weymann i can merge this if you add the tags and fix the build (please run mvn spotless:apply)

Reply:
Thx, done!